emails/management/commands/check_health.py

```diff
@@ -24,20 +25,29 @@
 class Command(BaseCommand):
     help = "Check that a healthcheck JSON file exists and is recent."
 
-    DEFAULT_MAX_AGE_SECONDS = 120
+    def help_message(self, text, setting_name):
```

Comment:
suggestion (non-blocking): can this be `help_message_for_setting` to make it more obvious what the 2nd argument is when calling this? E.g., `self.help_message_for_setting("Path to healthcheck JSON file.", "PROCESS_EMAIL_HEALTHCHECK_PATH")`